whatrecord/bin/iocmanager_loader.py

```diff
@@ -26,10 +26,19 @@ def build_arg_parser(parser=None):
         "configs", type=str, nargs="+", help="Configuration file location(s)"
     )
 
+    parser.add_argument(
+        "--limit", type=int, default=0, help="Limit the number of IOCs to output"
+    )
+
     return parser
 
 
-def main(configs) -> List[IocMetadata]:
+def main(configs: list[AnyPath], limit: int = 0) -> List[IocInfoDict]:
     iocs = get_iocs_from_configs(configs)
-    print(json.dumps(iocs, indent=4))
+    if limit:
+        dumped = json.dumps(iocs[-limit:], indent=4)
```

Comment:
Is there a reason why we slice starting from the end of the list here, and not from the start?
`iocs[-limit:]` vs `iocs[:limit]`

Reply:
I thought it might give me more recent IOCs for the purposes of testing ("instead of the 5 oldest IOCs, give me 5 recently created ones")

Reply:
Makes sense!